lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php

```diff
@@ -1648,10 +1347,7 @@ function classHasAttribute($attributeName)
  */
 function classHasStaticAttribute($attributeName)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$attributeName". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)